.github/workflows/build-test.yaml

```diff
@@ -49,7 +49,7 @@ on:
   workflow_dispatch: # allow manual trigger
 
 env:
-  dummy: 8                 # change to force cache invalidation
+  dummy: 6                 # change to force cache invalidation
```

Comment:
Can we try to just increase this number all the time please.

Reply:
Is it OK if I change it to `# increment to force cache invalidation`?